style.css

```diff
@@ -0,0 +1,313 @@
+  display: flex;
+}
+
+a {
+  text-decoration: none;
+}
+
+main {
+  position: relative;
+  background-color: #fff;
+  background-image: url("../img/mobile.png");
+  background-repeat: no-repeat;
+  margin: 1.625rem 0;
+  height: 36.697rem;
+  border-bottom-left-radius: 80px;
+}
+
+h1 {
+  height: 9.375rem;
+  width: 15rem;
+  margin: 3rem 1.2rem;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+main p {
+  width: 20.438rem;
+  margin: 0 1.5rem;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+
+  /* or 150% */
+
+  color: #344563;
+}
+
+h5 {
+  margin: 0.75rem 1.5rem;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: #7f8cff;
+  width: 7.313rem;
+  height: 1.5rem;
+}
+
+ul {
+  list-style-type: none;
+}
+
+main ul {
+  width: 9.75rem;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 1.5rem;
+}
+
+main li {
+  margin-left: 0;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  width: 375px;
+}
+
+.content {
+  margin-top: 248px;
+  margin-right: 16px;
+  margin-left: 16px;
+  margin-bottom: 12px;
+  height: 80px;
+}
+
+.content h4 {
+  width: 295px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 12px 0;
+}
+
+.grid-item {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  background-color: #fff;
+  color: #344563;
+  background-repeat: no-repeat;
+  background-position: 16px 16px;
+  border-radius: 16px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 16px;
+  width: 327px;
+  height: 552px;
+}
+
+.grid-item ul {
+  width: 203px;
+  height: 24px;
+  padding-left: 0;
+}
+
+.content ul {
+  list-style-type: disc;
+  justify-content: space-between;
+  width: 236px;
+  padding-left: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.content li {
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.ocontent li {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  margin-left: 16px;
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  padding: 6px 12px;
+  align-items: center;
+}
+
+.grid-item p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 12px 16px;
+}
+
+.ttt {
+  list-style-type: none;
+}
+
+ul .ttt {
+  color: #000;
+}
+
+button {
+  width: 124px;
```

Comment:
Kindly, I recommend you manage the dimensions of the buttons with padding instead of fixed measurements with width